documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -18,82 +21,121 @@ Want to help us further hone and improve it? link:/docs/contribute/[Learn how].
 toc::[]
 endif::community[]
 
-{prodname}'s SQL Server Connector can monitor and record the row-level changes in the schemas of a SQL Server database.
+The {prodname} SQL Server connector captures row-level changes that occur in the schemas of a SQL Server database.
+
+ifdef::product[]
+For details about the {prodname} SQL Server connector and its use, see following topics: 
+
+* xref:overview-of-debezium-sql-server-connector[]
+* xref:how-debezium-sql-server-connectors-work[]
+* xref:descriptions-of-debezium-sql-server-connector-data-change-events[]
+* xref:how-debezium-SQL Server-connectors-map-data-types[]
+* xref:setting-up-sql-server-for-use-with-the-debezium-sql-server-connector[]
+* xref:deploying-debezium-sql-server-connectors[]
+* xref:Monitoring {prodname} SQL Server connector performance[]
+* xref:how-debezium-postgresql-connectors-handle-faults-and-problems[]
+
+endif::product[]
+
+The first time that the {prodname} SQL Server connector connects to a SQL Server database or cluster, it takes a consistent snapshot of the schemas in the database.
+After the initial snapshot is complete, the connector continuously captures row-level changes for `INSERT`, `UPDATE`, or `DELETE` operations that are committed to tracked SQL Server databases. 
```

Comment:
@roldanbob Word `tracking` and `tracked` can be really confusing in SSQL Server context as SQL Server supports change data capture we rely upon and Change *Tracking* (https://docs.microsoft.com/en-us/sql/relational-databases/track-changes/about-change-tracking-sql-server?view=sql-server-ver15) which is similar but distinct feature